yotpo.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'faraday'
   gem.add_dependency 'typhoeus'
-  gem.add_dependency 'faraday_middleware', '>= 0.8', '< 0.10'
+  gem.add_dependency 'faraday_middleware', '~> 0.12.0'
```

Comment:
Can you change it back to '>= 0.8', '< 0.10' ? 
We have other gems that dependent on the old version of faraday_middleware